modules/virt-opting-out-workload-updates.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+[id="virt-opting-out-workload-updates_{context}"]
+= Opting out of automatic workload updates before you upgrade to {VirtProductName} {VirtVersion}
+
+Automatic workload updates are enabled by default as of {VirtProductName} {VirtVersion}. If you plan to disable this feature, you can do so _before_ upgrading by following this workaround.
+
+[NOTE]
+====
+If you already upgraded to {VirtProductName} {VirtVersion}, you can disable automatic workload updates by setting the value of `spec.workloadUpdateStrategy.workloadUpdateMethods` to `[]` in the `HyperConverged` CR.
+====
+
+.Prerequisites
+
+* You have not yet upgraded to {VirtProductName} {VirtVersion}.
+
+.Procedure
+
+. Annotate the `HyperConverged` CR by running the following command:
```

Comment:
I am a bit confused here, are we suggesting these be done before CNV upgrade but after OCP upgrade? Can this be made a bit clear please?